{"name": "test_example", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "@pytest.mark.flaky(reruns=5)\n    def test_example():\n        # import random\n        # assert random.choice([True, False])\n>       assert False\nE       assert False\n\ntest_rerun.py:14: AssertionError"}, "start": 1622387658446, "stop": 1622387658446, "uuid": "7b932c97-16e6-4e8e-b1d3-1e4e1ec70f3c", "historyId": "36599bcaff4a062c2f3cd1243bd12f09", "testCaseId": "dfbaabc1178922d2b3d9b33afee30ee7", "fullName": "test_rerun#test_example", "labels": [{"name": "tag", "value": "flaky(reruns=5)"}, {"name": "suite", "value": "test_rerun"}, {"name": "host", "value": "mozhenraodeMacBook-Air.local"}, {"name": "thread", "value": "7773-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_rerun"}]}